src/fides/api/util/consent_util.py

```diff
@@ -57,27 +57,32 @@
 
 
 def filter_privacy_preferences_for_propagation(
-    system: Optional[System], privacy_preferences: List[PrivacyPreferenceHistory]
-) -> List[PrivacyPreferenceHistory]:
+    system: Optional[System],
+    privacy_preferences: List[PrivacyPreferenceHistoryV2],
+) -> List[PrivacyPreferenceHistoryV2]:
     """Filter privacy preferences on a privacy request to just the ones that should be considered for third party
     consent propagation.
 
     Only applies to preferences saved for privacy notices here, not against individual TCF components.
     """
 
-    propagatable_preferences: List[PrivacyPreferenceHistory] = [
+    propagatable_preferences: List[PrivacyPreferenceHistoryV2] = [
         pref
         for pref in privacy_preferences
         if pref.privacy_notice_history
         and pref.privacy_notice_history.enforcement_level
         == EnforcementLevel.system_wide
-        and pref.preference != UserConsentPreference.acknowledge
+        and (
+            pref.preference
+            and pref.preference
+            in [UserConsentPreference.opt_in, UserConsentPreference.opt_out]
+        )
```

Comment:
Now that we have a new "tcf" preference we needed to update "filter_privacy_preferences_for_propagation" to look at just opt out/opt in, so the new TCF preference was excluded.